recipe/bld.bat

```diff
@@ -1,4 +1,4 @@
 mkdir "%PREFIX%"\lib
 mkdir "%SCRIPTS%"\
 copy "%SRC_DIR%\%PKG_NAME%-%PKG_VERSION%.jar" "%PREFIX%\lib\bfg.jar"
-echo java -jar %PREFIX%\lib\bfg.jar %%* >> "%SCRIPTS%\bfg.bat"
+echo java -jar ^%CONDA_PREFIX^%\lib\bfg.jar %%* >> "%SCRIPTS%\bfg.bat"
```

Comment:
If you leave `%PREFIX%` here, conda should automatically replace this with the correct path on installation.